x-pack/plugins/apm/server/lib/services/get_throughput.ts

```diff
@@ -25,6 +25,8 @@ interface Options {
   serviceName: string;
   setup: Setup & SetupTimeRange;
   transactionType: string;
+  start: number;
+  end: number;
```

Comment:
Why is this being passed in separately when it's already part of `setup`?

Reply:
This is neede becuase now I call [getThroughput](https://github.com/cauemarcondes/kibana/blob/apm-time-comparison-throughput-chart/x-pack/plugins/apm/server/routes/services.ts#L352-L359) twice one to fetch the current period, that uses the start and end from setup, and second to fetch the comparison data.

Reply:
Probably better to remove SetupTimerange here, to prevent code from getting start and end from the setup object.